src/components/autocomplete/demoRepeatMode/index.html

```diff
@@ -0,0 +1,41 @@
+<div ng-controller="DemoCtrl as ctrl" layout="column" ng-cloak>
+  <md-content class="md-padding">
+    <form ng-submit="$event.preventDefault()">
+      <p id="autocompleteDescription">
+        Use the <code>md-mode</code> attribute to fall back to ng-repeat for autocomplete entries.
+        Note that sizable lists can have performance implications for your application. Using modes
+        other than 'virtual' will require you to address those directly within your application.
+      </p>
+      <md-autocomplete
+          ng-disabled="ctrl.isDisabled"
+          md-no-cache="ctrl.noCache"
+          md-selected-item="ctrl.selectedItem"
+          md-search-text-change="ctrl.searchTextChange(ctrl.searchText)"
+          md-search-text="ctrl.searchText"
+          md-selected-item-change="ctrl.selectedItemChange(item)"
+          md-items="item in ctrl.querySearch(ctrl.searchText)"
+          md-item-text="item.display"
+          md-min-length="0"
+          placeholder="What is your favorite US state?"
+          aria-describedby="autocompleteDescription autocompleteDetailedDescription"
+          md-mode="standard">
+        <md-item-template>
+          <span md-highlight-text="ctrl.searchText" md-highlight-flags="^i">{{item.display}}</span>
+        </md-item-template>
+        <md-not-found>
+          No states matching "{{ctrl.searchText}}" were found.
+          <a ng-click="ctrl.newState(ctrl.searchText)">Create a new one!</a>
+        </md-not-found>
+      </md-autocomplete>
+      <br/>
+      <md-checkbox ng-model="ctrl.simulateQuery">Simulate query for results?</md-checkbox>
+      <md-checkbox ng-model="ctrl.noCache">Disable caching of queries?</md-checkbox>
+      <md-checkbox ng-model="ctrl.isDisabled">Disable the input?</md-checkbox>
```

Comment:
Let's get rid of these options. I'd like to see 2 autocompletes, side by side. One with standard and one with virtual.

Then we need an example of variable height suggestions. It could be in the standard mode autocomplete above or a separate autocomplete in this same demo.

For a data set, perhaps you could use the same JSON data from the custom template demo but add to it a description field that has a different length for each item.

Reply:
k. I've added descriptions using text from their github pages. What do you think?